cobald_tests/utility/concurrent/test_meta_runner.py

```diff
@@ -43,7 +43,8 @@ async def t_coroutine():
             assert bool(runner)
             runner.stop()
 
-    def test_run_subroutine(self):
+    @pytest.mark.parametrize("flavour", (threading,))
```

Comment:
The ``parametrize("flavour", ...)`` moves the ``for flavour in ...:`` out of the test to let ``pytest`` to report each case individually. This allows to check which flavour failed.

I've applied this to all tests, even those with only one flavour, for consistency.